src/docGenerator.ts

```diff
@@ -75,11 +76,14 @@ export class DocGenerator {
         parameters
       )
     })
-    return new Patterns(toolName, toolVersion, entries)
+
+    const completeEntryArray = entries.concat(extraPatternEntries)
+
+    return new Patterns(toolName, toolVersion, completeEntryArray)
   }
 
   generateDescriptionEntries(): DescriptionEntry[] {
-    return flatMap(this.rules, ([patternId, ruleModule]) => {
+    var descriptions = flatMap(this.rules, ([patternId, ruleModule]) => {
```

Comment:
This can be a `const`:
```suggestion
    const descriptions = flatMap(this.rules, ([patternId, ruleModule]) => {
```